src/Native.res

```diff
@@ -6,10 +6,20 @@ type themeColors = {
   border: string,
   notification: string,
 }
+
+type themeFonts = {
+  regular: ReactNative.Style.t,
+  medium: ReactNative.Style.t,
+  bold: ReactNative.Style.t,
+  heavy: ReactNative.Style.t,
+}
+
 type theme = {
   dark: bool,
   colors: themeColors,
+  fonts: themeFonts,
```

Comment:
This is `react-navigation` v7 only, I'll prefer keeping it on dedicated PR

Reply:
I can do a separate PR for it. But while the additional prop does no harm in RN 6, RN 7 will crash if it is missing. So I'd really like to get this in soon...

Reply:
But we can't release a v7 without doing the upgradings : https://reactnavigation.org/docs/upgrading-from-6.x

And we can't force v6 users to create a fake theme fonts…